### DISCUSSION TITLE
Redirect new syntax

### DISCUSSION THREAD

**Discussion by venkymylaram** [remix-run/react-router#11674] [Q&A]
How to write new syntax for redirect in v6

      <ApplicationRoute
        path={"*"}
        element={<Navigate to={MANAGE_REPORTS_ROUTES.ELIGIBILITY_FILTER} />}
      />
      
      if i use * its working for route mismatch,how to  write redirect from specific path

**Comment by sergiodxa** [comment]
In the route loader return a redirect response
